assets/js/calendar.js

```diff
@@ -0,0 +1,69 @@
+const CALENDAR_ID = 't8iphvpqadetnf7ccpb2c7pnjk@group.calendar.google.com'
+const CALENDAR_KEY = 'AIzaSyBplGV41n0rSD_NK612JhLTxePXXeyo0iE'
+
+const SMALL_DEVICE_WIDTH = 600
+
+const DATE_24HR_FORMAT = {
+  hour: '2-digit',
+  minute: '2-digit',
+  omitZeroMinute: false,
+  hour12: false
+}
+
+let calendar
+let calendarElement
+
+document.addEventListener('DOMContentLoaded', () => {
+  calendarElement = document.querySelector('#calendar')
+
+  let defaultView = 'dayGridMonth'
```

Comment:
Looks like this might be the bit?

Reply:
581c83188e5aa233ae287bb822612f57cc25d770 resolves this